src/KeyManager.ts

```
@@ -372,6 +374,32 @@ export class KeyManager {
       );
     }
 
+    if (authServerSigningKey) {
```

Shouldn't we make this a requirement instead and raise an exception when not provided?

---

Good call.